bridges/SymfonyCastsBridge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class SymfonyCastsBridge extends BridgeAbstract {
+	const NAME = 'SymfonyCasts Bridge';
+	const URI = 'https://symfonycasts.com/';
+	const DESCRIPTION = 'Follow new updates on symfonycasts.com';
+	const MAINTAINER = 'No maintainer';
+	const CACHE_TIMEOUT = 3600;
+
+	public function collectData() {
+		$html = getSimpleHTMLDOM('https://symfonycasts.com/updates/find');
+		$dives = $html->find('div');
+
+		/* @var simple_html_dom $div */
+		foreach ($dives as $div) {
+			//var_dump($div);die();
+			//dump_html_tree($div);die();
+			$id = $div->getAttribute('data-mark-update-id-value');
+			$type = $div->find('h5');
+			if(is_array($type)) {
+				$type = $type[0];
+			}
+			$title = $div->find('span');
+			if(is_array($title)) {
+				$title = $title[0];
+			}
+
+			$dateString = $div->find('h5.font-gray');
+			if(is_array($dateString)) {
+				$dateString = $dateString[0];
+			}
+
+			$href = $div->find('a');
+			if(is_array($href)) {
+				$href = $href[0];
+			}
+			$url = 'https://symfonycasts.com' . $href->getAttribute('href');
+
+			$item = array(); // Create an empty item
+			$item['uid'] = $id;
+			$item['title'] = $title->innertext;
+			$item['timestamp'] = $dateString->innertext;
+			$item['content'] = $type->content . '<a href="' . $url . '">' . $title . '</a>';
```

Comment:
`$type->content` returns boolean false. Use `$type->plaintext` instead, it returns what you want.
```suggestion
			$item['content'] = $type->plaintext. '<a href="' . $url . '">' . $title . '</a>';
```